src/UnderlineNav2/UnderlineNav.test.tsx

```diff
@@ -79,34 +113,89 @@ describe('UnderlineNav', () => {
     fireEvent.keyPress(item, {key: 'Enter', code: 13, charCode: 13})
     expect(onSelect).toHaveBeenCalledTimes(2)
   })
-  test('respect counter prop', () => {
-    const {getByText} = render(
-      <UnderlineNav label="Test nav" align="right">
-        <UnderlineNav.Item counter={8} selected>
-          Item 1
-        </UnderlineNav.Item>
-        <UnderlineNav.Item>Item 2</UnderlineNav.Item>
-        <UnderlineNav.Item>Item 3</UnderlineNav.Item>
-      </UnderlineNav>
-    )
-    const item = getByText('Item 1').closest('a')
-    const counter = item?.getElementsByTagName('span')[2]
+  it('respects counter prop', () => {
+    const {getByText} = render(<ResponsiveUnderlineNav />)
+    const item = getByText('Issues').closest('a')
+    const counter = item?.getElementsByTagName('span')[3]
     expect(counter?.className).toContain('CounterLabel')
-    expect(counter?.textContent).toBe('8')
+    expect(counter?.textContent).toBe('120')
   })
-  test('respect loadingCounters prop', () => {
-    const {getByText} = render(
-      <UnderlineNav label="Test nav" loadingCounters={true}>
-        <UnderlineNav.Item selected counter={4}>
-          Item 1
-        </UnderlineNav.Item>
-        <UnderlineNav.Item>Item 2</UnderlineNav.Item>
-        <UnderlineNav.Item>Item 3</UnderlineNav.Item>
-      </UnderlineNav>
-    )
-    const item = getByText('Item 1').closest('a')
+  it('respects loadingCounters prop', () => {
+    const {getByText} = render(<ResponsiveUnderlineNav loadingCounters={true} />)
+    const item = getByText('Actions').closest('a')
     const loadingCounter = item?.getElementsByTagName('span')[2]
     expect(loadingCounter?.className).toContain('LoadingCounter')
     expect(loadingCounter?.textContent).toBe('')
   })
 })
+
+describe.skip('UnderlineNav when overflow occurs on fine pointer devices', () => {
```

Comment:
🆘 Help Needed 🆘 

I cannot seem to make these tests work. Because jsdom doesn't have the ability to calculate where elements will be visually laid out as a result of CSS [ref](https://github.com/jsdom/jsdom#unimplemented-parts-of-the-web-platform) the list items width return 0 so that I cannot really test the responsive behaviour. I tried to add `Object.defineProporty` above but that doesn't seem to do the trick. I would appreciate any help on to test these responsive behaviour i.e `More` menu button visibility.

Reply:
Just a context here that we are discarding writing unit test for this complex visual cases. I'll be writing interaction tests for it on a different PR 🥳